src/main/java/ro/msg/learning/shop/entity/Location.java

```diff
@@ -0,0 +1,23 @@
+package ro.msg.learning.shop.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+
+@Data
+@Entity
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class Location extends BaseEntity {
+
+    private String name;
+    private String address_country;
+    private String address_city;
+    private String address_county;
+    private String address_street_address;
```

Comment:
You can create a separate Entity named Address with these fields. In this way you remove duplicates because the fields are used also in Order entity.